google-account-plugin/resources/META-INF/plugin.xml

```diff
@@ -57,6 +57,8 @@
                         serviceImplementation="com.google.cloud.tools.intellij.DefaultAccountPluginConfigurationService"/>
     <applicationService serviceInterface="com.google.cloud.tools.intellij.login.GoogleLoginService"
                         serviceImplementation="com.google.cloud.tools.intellij.login.IntellijGoogleLoginService"/>
+    <applicationService serviceInterface="com.google.cloud.tools.intellij.analytics.IdeaUsageTracker"
```

Comment:
The plan is to do this in multiple steps to lessen the review burden. The first step is to introduce and set up new facilities. The second is to change exist code to use it. The last is to remove old code, which is pretty large and complex.